packager/media/formats/mp2t/ac3_header.cc

```diff
@@ -104,13 +117,15 @@ size_t Ac3Header::GetHeaderSize() const {
 }
 
 size_t Ac3Header::GetFrameSize() const {
-  const size_t kNumFscode = arraysize(kAc3SampleRateTable);
-  DCHECK_LT(fscod_, kNumFscode);
-  DCHECK_LT(frmsizecod_, arraysize(kFrameSizeCodeTable));
-  // The order of frequencies are reversed in |kFrameSizeCodeTable| compared to
-  // |kAc3SampleRateTable|.
-  const int index = kNumFscode - 1 - fscod_;
-  return kFrameSizeCodeTable[frmsizecod_][index] * 2;
+  return CalcFrameSize(fscod_, frmsizecod_);
+}
+
+size_t Ac3Header::GetFrameSizeWithoutParsing(const uint8_t* data,
+                                             size_t num_bytes) const {
+  DCHECK_GT(num_bytes, static_cast<size_t>(4));
+  uint8_t fscod = (data[4] & 0xc0) >> 6;
```

Comment:
nit: can be replaced with

uint8_t fscod = data[4] >> 6;

Reply:
Ah! Fair enough!